README.md

```diff
@@ -22,3 +22,6 @@ zigpy works with separate radio libraries which can each interface with multiple
   - [ConBee II (a.k.a. ConBee 2) USB adapter from Dresden-Elektronik](https://shop.dresden-elektronik.de/conbee-2.html)
   - [ConBee](https://www.dresden-elektronik.de/conbee/) USB adio adapter from [Dresden-Elektronik](https://www.dresden-elektronik.de)
   - [RaspBee](https://www.dresden-elektronik.de/raspbee/) GPIO radio adapter from [Dresden-Elektronik](https://www.dresden-elektronik.de)
+  
+  **Related projects:**
+  ZHA exception and deviation handling in Home Assistant relies on on the third-party [ZHA Device Handlers](https://github.com/dmulcahey/zha-device-handlers) project. Zigbee devices that deviate from or do not fully conform to the standard specifications set by the [Zigbee Alliance](https://www.zigbee.org) may require the development of custom [ZHA Device Handlers](https://github.com/dmulcahey/zha-device-handlers) (ZHA custom quirks handler implementation) to for all their functions to work properly with the ZHA component in Home Assistant. These ZHA Device Handlers for Home Assistant can thus be used to parse custom messages to and from Zigbee devices. The custom quirks implementations for zigpy implemented as ZHA Device Handlers for Home Assistant are a similar concept to that of [Hub-connected Device Handlers for the SmartThings Classics platform](https://docs.smartthings.com/en/latest/device-type-developers-guide/) as well as that of [Zigbee-Shepherd Converters as used by Zigbee2mqtt](https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html), meaning they are each virtual representations of a physical device that expose additional functionality that is not provided out-of-the-box by the existing integration between these platforms.
```

Comment:
> ZHA exception and deviation handling

I'm not entirely happy with this phrasing, as word "excpeption" makes me think about Python exceptions. Would `Non ZHA compliant device behavior handling in Home Assistant ...` sound better?

Reply:
I think both words are correct, see https://wikidiff.com/exception/deviation Reason for including both words even though they mean the same is so more non-native English speakers might at least grasp the meaning when first seeing the sentence.

Reply:
I see your point. I don't have many problem with the "deviation" word, but "exception" should be removed.  IMO `Non compliant device behavior` does describe it more precisely, while `ZHA Exception` leaves more room for confusion.

Reply:
OK, I have now removed the word "exception" and rephrased.